src/Hl7.Fhir.Specification/Specification/Source/CommonZipSource.cs

```diff
@@ -22,7 +21,7 @@ namespace Hl7.Fhir.Specification.Source
     /// <summary>Reads FHIR artifacts (Profiles, ValueSets, ...) from a ZIP archive. Thread-safe.</summary>
     /// <remarks>Extracts the ZIP archive to a temporary folder and delegates to the <see cref="CommonDirectorySource"/>.</remarks>
     [DebuggerDisplay(@"\{{DebuggerDisplay,nq}}")]
-    public class CommonZipSource : ISummarySource, IConformanceSource, IArtifactSource, IResourceResolver, IAsyncResourceResolver
+    public class CommonZipSource : ISummarySource, IArtifactSource, IResourceResolver, IAsyncResourceResolver
```

Comment:
This can now implement `ICommonConformanceSource`

Reply:
Yes, indeed. I solved that.